accord-core/src/test/java/accord/utils/Gens.java

```diff
@@ -112,8 +113,18 @@ public static <T> Gen<T> pick(Map<T, Integer> values)
     {
         if (values == null || values.isEmpty())
             throw new IllegalArgumentException("values is empty");
+        // if 2 values have the same weight we need some way to tie-break, but that isn't always possible...
```

Comment:
if no keys have the same value then we are fine, but there are cases where the values are randomized for each test... so figured its still best to block to specific classes just to be safe and find this issue earlier